internal/utils.go

```diff
@@ -326,23 +329,31 @@ func getPolicyTemplates(policyConf *types.PolicyConfig) ([]map[string]interface{
 // When index is 0, it won't attach any number. When useManifestName is true,
 // it returns the policyConf.Manifests[manifestGroupIndex].Name otherwise,
 // It returns policyConf.Name + index
-func getConfigurationPolicyName(policyConf *types.PolicyConfig, manifestGroupIndex, policyNum,
-	manifestIndex int, useManifestName bool,
+func getConfigurationPolicyName(policyConf *types.PolicyConfig, manifestGroupIndex int,
+	usedPolicyNameNum *int, usedManifestNameNum *int,
```

Comment:
(optional) Combine the arg types; Use proposed name
```suggestion
	policyNameCounter, manifestNameCounter *int,
```